pkg/agent/proxy/types/groupcounter.go

```diff
@@ -26,15 +26,14 @@ import (
 
 // GroupCounter generates and manages global unique group ID.
 type GroupCounter interface {
-	// Get generates a global unique group ID for a specific service.
-	// If the group ID of the service has been generated, then return the
-	// prior one. The bool return value indicates whether the groupID is newly
-	// generated.
+	// AllocateNotExist generates a global unique group ID for a Service if the group ID has not been generated, then
+	// return the group ID (new allocated or already allocated).
+	AllocateNotExist(svcPortName k8sproxy.ServicePortName, isEndpointsLocal bool) binding.GroupIDType
```

Comment:
```suggestion
	AllocateIfNotExist(svcPortName k8sproxy.ServicePortName, isEndpointsLocal bool) binding.GroupIDType
```
AllocateNotExist means something else.

Reply:
Updated.